plugin.go

```diff
@@ -200,6 +200,11 @@ func commandBuild(build Build) *exec.Cmd {
 	if build.Compress {
 		args = append(args, "--compress")
 	}
+    if build.Cache {
+        args = append(args, "--pull=false")
+    } else {
+        args = append(args, "--pull=true")
+    }
```

Comment:
Doooh.. now I feel silly! I knew there was a way of doing this without negating the option, thanks!